### ISSUE TITLE
[FEATURE] Support changing BatchNorm layer in all backbones

### ISSUE THREAD

**Issue by triwahyuu** [nodefluxio/vortex#61]
**Is your feature request related to a problem? Please describe.**
There some use cases for changing the BatchNorm layers in the backbones, especially when using transfer learning it would be more efficient to use a frozen batchnorm in the backbone.

**Describe the solution you'd like**
support changing BatchNorm layer in all backbones, using `norm_layer` argument like those in [resnet torchvision](https://github.com/pytorch/vision/blob/ac3ba94440af8eaafc3c21353006b5d0fc9aed2a/torchvision/models/resnet.py#L124)
